app/assets/stylesheets/govuk_publishing_components/components/_contents-list.scss

```diff
@@ -84,12 +84,6 @@
     margin-left: 0;
   }
 
-  // Put indentation back where we use list style types
-  .gem-c-contents-list__list-item--dashed {
```

Comment:
Good improvement. Is it also worth setting the link colour on the contents list to the print/black?

On that, I've been thinking about a general approach and wondering if that's something we can have in more general print styles. Not for this PR, but we could have a general class to apply to links where we want them to print not in the link colour. Separating it out like this would help clarify the intent behind it, and limit the number of times we have to repeat this across the components.

```CSS
.print-link-style {
  color: $govuk-print-text-colour !important;
}
```

```HTML
<a href="#" class="govuk-link">Don't care what this looks like when printed</a>
<a href="#" class="govuk-link print-link-style">Do care what this looks like when printed</a>
```

Reply:
I've added in a colour rule to force the content list links to print in black. For your other comment I've copied this out as a note into our backlog ticket as something to look at.